src/Domain/Fixture/FixtureCustomizer.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace Acquia\Orca\Domain\Fixture;
+
+use Acquia\Orca\Helper\Filesystem\FinderFactory;
+use Acquia\Orca\Helper\Filesystem\FixturePathHandler;
+use Acquia\Orca\Options\FixtureOptions;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * Customizes fixtures.
+ */
+class FixtureCustomizer {
+
+  /**
+   * The Finder Factory.
+   *
+   * @var \Acquia\Orca\Helper\Filesystem\FinderFactory
+   */
+  private $finderFactory;
+
+  /**
+   * The Filesystem service.
+   *
+   * @var \Symfony\Component\Filesystem\Filesystem
+   */
+  private $filesystem;
+
+  /**
+   * The path handler service for the fixture.
+   *
+   * @var \Acquia\Orca\Helper\Filesystem\FixturePathHandler
+   */
+  private $fixturePathHandler;
+
+  /**
+   * The output variable.
+   *
+   * @var \Symfony\Component\Console\Output\OutputInterface
+   */
+  private $output;
+
+  /**
+   * Constructs an instance.
+   *
+   * @param \Acquia\Orca\Helper\Filesystem\FinderFactory $finderFactory
+   *   The finder factory.
+   * @param \Symfony\Component\Filesystem\Filesystem $filesystem
+   *   The filesystem service.
+   * @param \Acquia\Orca\Helper\Filesystem\FixturePathHandler $fixturePathHandler
+   *   The path handler for fixture.
+   * @param \Symfony\Component\Console\Output\OutputInterface $output
+   *   The output variable.
+   */
+  public function __construct(FinderFactory $finderFactory,
+  Filesystem $filesystem,
+    FixturePathHandler $fixturePathHandler,
+    OutputInterface $output) {
+    $this->finderFactory = $finderFactory;
+    $this->filesystem = $filesystem;
+    $this->fixturePathHandler = $fixturePathHandler;
+    $this->output = $output;
+  }
```

Comment:
I've noticed this odd indentation on your multi-line function declarations lately, @secretsayan. You might want to check your PhpStorm code formatting settings.

```suggestion
  public function __construct(
      FinderFactory $finderFactory,
      Filesystem $filesystem,
      FixturePathHandler $fixturePathHandler,
      OutputInterface $output
  ) {
      $this->finderFactory = $finderFactory;
      $this->filesystem = $filesystem;
      $this->fixturePathHandler = $fixturePathHandler;
      $this->output = $output;
  }
```